x-pack/plugins/infra/common/alerting/metrics/alert_link.ts

```diff
@@ -46,37 +49,66 @@ export const getInventoryViewInAppUrl = (
   }
 
   const nodeTypeField = `${ALERT_RULE_PARAMETERS}.nodeType`;
-  const nodeType = inventoryFields[nodeTypeField];
-  let inventoryViewInAppUrl = '/app/metrics/link-to/inventory?';
+  const nodeType = inventoryFields[nodeTypeField] as InventoryItemType;
+  const hostName = inventoryFields[HOST_FIELD];
+  let viewInAppUrl = '/app/metrics/link-to/inventory?';
```

Comment:
I think you could move this outside and make this a const.
```suggestion
  const viewInAppUrl = '/app/metrics/link-to/inventory?';
```